eth/api.go

```diff
@@ -759,3 +759,26 @@ func (api *ScrollAPI) GetSkippedTransactionHashes(ctx context.Context, from uint
 
 	return hashes, nil
 }
+
+// ResetSkippedTransactionsTraces reset the traces for skipped txs stored in db, and returns the list of reset transaction hashes.
```

Comment:
```suggestion
// ResetSkippedTransactionsTraces resets the traces for skipped txs stored in db, and returns the list of reset transaction hashes.
```

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/501/commits/1b7db0024f6f413d9958c1cfcbf281abdf1db45b